index.js

```diff
@@ -194,6 +246,28 @@ class Functions {
     return string.length > length ? string.slice(0, length) + (placeholder ? placeholder.toString() : '...') : string;
   }
 
+  /**
+   * Decancers a string. Reduces everything from zalgos, fancy characters, etc. to alphanumeric characters.
+   * NOTE: The string returned will ALWAYS be in lowercase.
+   * @param {string} text The cancerous string to decancer.
+   * @returns {string} The cleaned string. This may break certain things like special characters. Only use for certain purposes like filtering/censoring.
+   */
+  decancer(text) => {
```

Comment:
```suggestion
  decancer(text) {
```
Unnecessary arrow expression.